### ISSUE TITLE
Polymer 2.0?

### ISSUE THREAD

**Issue by rene-lindner-isw** [PolymerElements/iron-signals#25]
Will there be an 2.0 Version of this element or this is deprecated?

**Comment by ernsheong** [comment]
I made a hard "fork" for Polymer 2 support (I also didn't like the plural name :P). Check it out to see if it works for you: https://github.com/ernsheong/lite-signal

**Comment by ThilinaTCH** [comment]
Any news on updating this element to Polymer2 or hybrid?
If this is deprecated what would be the alternative to use for?

**Comment by ernsheong** [comment]
@ThilinaTCH did you read the previous comment?

**Comment by ThilinaTCH** [comment]
@ernsheong Yeah but seems it changes the syntax of already existing, so more like a new element?
